### PR TITLE
Countrylist

### PR DESCRIPTION
**Pull request by Carolin-Weitkamp**
a lot of styling. Styles for day and night.

### CODE REVIEW

File: components/LoadingSide/LoadingSide.js [hunk 1]
```diff
@@ -123,37 +109,26 @@ const Loading6 = styled.div`
...
+  border-bottom: ${({ theme }) => theme.border};
   background: var(--purple-gradient-rl);
   @media screen and (min-width: 700px) {
     grid-column: col-start 6 / span 1;
     grid-row: 1 / 14;
-    border-left: 0.3vw solid var(--lightgreen);
-    border-right: 0.3vw solid var(--lightgreen);
-    border-bottom: 0.3vw solid var(--lightgreen);
   }
 `;
```
Comment (NikRoe) [inline components/LoadingSide/LoadingSide.js:134]: the same as above: do you need 9 different styled components or would one default with 8 extension make more sense?

File: components/Results/Results.js [hunk 2]
```diff
@@ -214,13 +184,8 @@ const Result8 = styled.div`
...
+  border-left: ${({ theme }) => theme.border};
+  border-right: ${({ theme }) => theme.border};
+  border-bottom: ${({ theme }) => theme.border};
   background: var(--purple-gradient-rl);
-  @media screen and (min-width: 700px) {
-    border-left: 0.3vw solid var(--lightgreen);
-    border-right: 0.3vw solid var(--lightgreen);
-    border-bottom: 0.3vw solid var(--lightgreen);
-  }
 `;
```
Comment (NikRoe) [inline components/Results/Results.js:191]: as above :)

File: pages/index.js [hunk 3]
```diff
@@ -110,68 +103,51 @@ const Landing4 = styled.div`
...
+  border-bottom: ${({ theme }) => theme.border};
   background: var(--purple-gradient-rl);
-  @media screen and (min-width: 700px) {
-    border-left: 0.3vw solid var(--lightgreen);
-    border-right: 0.3vw solid var(--lightgreen);
-    border-bottom: 0.3vw solid var(--lightgreen);
-  }
 `;
 
 // const PictureStyle = styled.div`
```
Comment (NikRoe) [inline pages/index.js:153]: you could remove that comment

File: pages/index.js [hunk 4]
```diff
@@ -110,68 +103,51 @@ const Landing4 = styled.div`
...
+  border-left: ${({ theme }) => theme.border};
+  border-right: ${({ theme }) => theme.border};
+  border-bottom: ${({ theme }) => theme.border};
   background: var(--purple-gradient-rl);
-  @media screen and (min-width: 700px) {
-    border-left: 0.3vw solid var(--lightgreen);
-    border-right: 0.3vw solid var(--lightgreen);
-    border-bottom: 0.3vw solid var(--lightgreen);
-  }
 `;
```
Comment (NikRoe) [inline pages/index.js:151]: as above :)

**Review by NikRoe — APPROVED** [review]
looks like some great work! 🙌

you might want to think about extending your components instead of writing up to 9 or 10 single styled components.

File: components/GlobalStyle.js [hunk 5]
```diff
@@ -15,11 +15,14 @@ body {
 }
 :root {
     font-family: 'Roboto', sans-serif;
-    --lightgreen: #61d286;
+    --lightgreen: hotpink;
```
[the remark this answers is not in the document]
Comment (Carolin-Weitkamp) [reply components/GlobalStyle.js:18]: of corse hotpink is the best - i´ll defenetly change it :)

**Pull request merged**